app/models/supplejack_api/concept.rb

```diff
@@ -6,8 +6,6 @@ class Concept
     include Mongoid::Document
     include Mongoid::Attributes::Dynamic
 
-    store_in client: 'api', collection: 'concepts'
```

Comment:
removing this will imply the model to use the concepts collection on the [worker_production DB](https://github.com/DigitalNZ/supplejack_worker/blob/master/config/mongoid.yml#L59), is that what you want?

Reply:
Yeah, I believe that is the intention. We don't want Concepts to be using the API database.

Reply:
All good 👍 . My question was are we removing concepts.